api/command_help_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package api
+
+import (
+	"github.com/mattermost/platform/model"
+	"testing"
+	"github.com/mattermost/platform/utils"
+)
+
+func TestHelpCommand(t *testing.T) {
+	th := Setup().InitBasic()
+	Client := th.BasicClient
+	channel := th.BasicChannel
+
+	rs := Client.Must(Client.Command(channel.Id, "/help ")).Data.(*model.CommandResponse)
```

Comment:
@cpanato 
what does 'v2 api' mean?
is it Client4.executeCommand() what you said?